dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,258 @@
+like this:
+
+```sh
+$ sudo visudo
+...
+##
+# User specification
+##
+
+# root and users in group wheel can run anything on any machine as any user
+root        ALL = (ALL) ALL
+%admin      ALL = (ALL) ALL
+username    ALL = (ALL) ALL
+```
+
+Press `Esc` followed by `:wq` to save and quit `visudo`.
+
+### Install XCode Command Line Tools ###
+
+Download XCode and Accept the License Agreement, then install the XCode
+Command Line Tools.
+
+```bash
+sudo xcodebuild -license
+xcode-select --install
+```
+
+### Docker ###
+
+Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+Install it by double-clicking the downloaded `dmg` file and dragging to the
+`Applications` folder.
+
+### iTerm2 (recommended) ###
+
+To set up iTerm2 - a MacOS terminal replacement with some nifty features, visit
+the [iTerm2 download page](https://www.iterm2.com/downloads.html) to get the
+latest stable release. Install it by double-clicking the downloaded `dmg` file
+and dragging to the `Applications` folder.
+
+Once installed, run the iTerm2 app and select the menu option
+`iTerm2 > Make iTerm2 Default Term`. NOTE: You may also need to set this
+setting as your Standard user account.
+
+### VPN Configuration ###
+
+Once you're set up with the lab (CAL) VPN connection, we recommend you set up
+split-tunneling. Instructions and a generator script are located in the
+[vpn-config-gen repository](https://github.com/cisagov/vpn-config-gen).
+
+## User Environment Setup ##
+
+For this section and following sections, log back in as your standard user.
+
+### Automatic Package Installation (recommended) ###
+
+To set up a dev environment via the [CISA `laptop` script](https://github.com/cisagov/laptop/),
+execute the following in your terminal:
+
+```sh
+bash <(curl --silent https://raw.githubusercontent.com/cisagov/laptop/master/laptop)
+```
+
+This will `curl` the current install files and script. You can specify
+additional packages to install by creating a `Brewfile.local`, as described in
+the Readme for the `laptop` repo where an example is provided.
+
+### Manual Package Installation (alternative) ###
+
+Open Terminal and install `brew` per the instructions from
+[Homebrew](https://brew.sh). You will probably have to do this with `sudo`.
+
+```sh
+/bin/bash -c "$(curl --fail --silent --show-error --location https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+
+After installing Brew, you'll want to install other useful packages. We
+recommend installing all the packages specified in the [CISA `laptop` script repository](https://github.com/cisagov/laptop/).
+
+### Environment configuration ###
+
+For portability between computers and environments, you may want to switch
+from the usual dotfiles setup to using `stow` to be able to use a personal
+repository and then map your dotfiles to that directory. This makes it much
+easier to sync and set up.
+
+Using a `stow` based setup also allows for some nifty features like
```

Comment:
`stow` based :arrow_right: `stow`-based

Reply:
Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4